qualification/pytest-jenkins-lab.ini

```diff
@@ -0,0 +1,23 @@
+[pytest]
+tester = Jenkins
+asyncio_mode = auto
+asyncio_default_fixture_loop_scope = session
+master_controller_host = lab5.sdp.kat.ac.za
+master_controller_port = 5001
+prometheus_url = http://lab5.sdp.kat.ac.za:9090
+product_name = jenkins_qualification_cbf
+interface = enp193s0f0np0
+interface_gbps = 90
+use_ibv = true
+# This core ordering distributes load across the CCDs of an Epyc 7402P
+cores = 0 6 12 18 1 7 13 19 2 8 14 20 3 9 15 21 4 10 16 22 5 11 17 23
```

Comment:
I've been meaning to ask how you obtain this list?

Reply:
There are four CCDs and six cores per CCD, numbered contiguously. So `0 6 12 18` is the first core of each CCD. Then `1 7 13 19` is the second core of each CCD etc.